packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -169,12 +169,17 @@ export default class ConfigParser {
         this._config = deepmerge(this._config, object) as TestrunnerOptionsWithParameters
 
         /**
-         * overwrite config specs that got piped into the wdio command
+         * overwrite config specs that got piped into the wdio command,
+         * also adhering to the wdio-prefixes from a capability
          */
-        if (object.specs && object.specs.length > 0) {
+        if (object['wdio:specs'] && object['wdio:specs'].length > 0) {
```

Comment:
This is why the build is failing, `object` are not capabilities here. We should revert these changes.

Reply:
hmmm, but if the check for `wdio:{specs,exclude}` is removed, we don't get the merged config and the tests (e.g.) `https://github.com/webdriverio/webdriverio/pull/11992/files#diff-e81dd1eb85b53f80e2f3860c68a8712a3b1bc2fa846f47909facc4637baf17d1R616` fail.

Above is of course directly connected to the `MergeConfig` type def:
```ts
interface MergeConfig extends Omit<Partial<TestrunnerOptionsWithParameters>, 'specs' | 'exclude'> {
    specs?: Spec[]
    'wdio:specs'?: Spec[]
    exclude?: string[]
    'wdio:exclude'?: string[]
}
```
What to do? 😄

Reply:
Oh ok, thanks for the call out. Let's leave it it as is then.